### ISSUE TITLE
Add learning rate scheduler for models

### ISSUE THREAD

**Issue by alzaia** [mila-aia/image_classification_simulation#74]
Select the most suitable LR scheduler for each of the models (schedulers available [here](https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate).

How to add it: https://pytorch-lightning.readthedocs.io/en/stable/common/optimization.html#learning-rate-scheduling.
